circle.yml

```diff
@@ -160,7 +160,15 @@ jobs:
            name: Test
            command: |
              ulimit -c 0
-             make distcheck
+             # Let's trick git.
+             #
+             # "git" running in "make distcheck" may refer $(pwd)/.git of THIS context.
+             # On the other hand, we want to run the test cases in an environment separated
+             # from our git repo.
+             #
+             mkdir __NO_GIT__
+             make distcheck GIT_DIR=__NO_GIT__
```

Comment:
I don't see the make variable GIT_DIR in Makefiles. Where is it defined?

I think checking with BUILD_IN_GIT_REPO in testing.mk should be enough.

Reply:
```
--git-dir=<path>
           Set the path to the repository (".git" directory). This can also be controlled by setting the GIT_DIR environment variable.
```

Reply:
~This change is for reproducing the failure locally.~ This change is for reproducing the failure at fedora34_distcheck CI/CD environemnet.